src/sentry/static/sentry/app/views/usageStats/index.tsx

```diff
@@ -46,7 +43,16 @@ class OrganizationStats extends React.Component<Props, State> {
   };
 
   get dataCategoryName() {
-    return capitalize(this.state.dataCategory);
+    switch (this.state.dataCategory) {
+      case DataCategory.ERRORS:
+        return t('Errors');
+      case DataCategory.TRANSACTIONS:
+        return t('Transactions');
+      case DataCategory.ATTACHMENTS:
+        return t('Attachments');
+      default:
```

Comment:
this page won't have sessions but they are another DataCategory that the route could potentially return at some point

Reply:
You could make an Enum/Map of these names and do something like `categoryNames[dataCategory] ?? t('Unknown Data Category')` to golf this down.

Reply:
That's fine for now, we are defining the options in `CHART_OPTIONS_DATACATEGORY` so the users will be unable to select sessions for now.